### PR TITLE
Add fip to header

### PR DESCRIPTION
**Pull request by amazingphilippe**
# Summary | Résumé

After demoing this a few times, the fip police came gently knocking. following this guidance, even internal pages must have the fip, so I'm proposing some changes:

image
image

# Test instructions | Instructions pour tester la modification

Test PR review app, or locally if we don't have one.
Check for any weird stuff happening on different viewport sizes.

### CODE REVIEW

**Review by andrewleith — COMMENTED** [review]
Looks good. Its too bad we  have to give up vertical space to add this as it was cleaner before without it, but I get it.

Should we be adding the wordmark, too?

**Comment by amazingphilippe** [comment]
> Should we be adding the wordmark, too?

Oh yeah! Good call, totally forgot about this one. I'll add it in!

File: src/index.html [hunk 1]
```diff
@@ -175,7 +175,11 @@ <h1 class="text-xl/7 font-heading font-[375]" data-i18n="site_title">
           <div id="history" class="dark:text-white" data-testid="history"></div>
         </div>
       </section>
-      <section class="bg-blue-900">
+      <section
+        class="bg-blue-900"
+        aria-label="Go to GC Notify"
+        data-i18n="back_button"
```
Comment (andrewleith) [inline src/index.html:181]: Probably a copy and paste issue: this is causing the i18n library to replace the contents of this element with the localized string

File: src/index.html [hunk 2]
```diff
@@ -186,89 +190,95 @@ <h1 class="text-xl/7 font-heading font-[375]" data-i18n="site_title">
         </div>
       </section>
     </main>
-    <footer class="bg-gray-100">
+    <footer
+      class="bg-gray-100"
+      aria-label="Go to GC Notify"
+      data-i18n="back_button"
```
Comment (andrewleith) [inline src/index.html:196]: Same issue here.  Remove these 2 `data-i18n` attributes and it should be good to go.

**Review by andrewleith — CHANGES_REQUESTED** [review]
Just a couple of wayward `data-i18n` attributes to remove.

File: src/index.html (hunk 1 above)
Comment (amazingphilippe) [reply src/index.html]: ugh, I know what happened, I was hoping it would target the aria-label, but I definitely doesn't!

File: src/js/i18n.js [hunk 3]
```diff
@@ -102,13 +167,24 @@
...
+          console.error("Missing translation for " + key);
+        } else {
+          //If translation applies to a specific attr, set it, else set innerHTML
+          if (STRINGS[key].attr) {
+            el.setAttribute(STRINGS[key].attr, STRINGS[key][lang]);
+          } else {
+            el.innerHTML = STRINGS[key][lang];
+          }
+        }
+      });
```
Comment (andrewleith) [inline src/js/i18n.js:187]: I skipped over this the first time around.  Very nice extension of this module to handle attributes ❤️

**Review by andrewleith — APPROVED** [review]
Nice work hunting down that HTML validation issue! 🥇

**Pull request merged**